targets/chrome-common

```diff
@@ -26,26 +26,30 @@ REQUIRES='x11'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-# Only download if it's not already installed, since it auto-updates
-pkg="google-chrome-$CHANNEL"
-if [ -n "`list_uninstalled '' "$pkg"`" ]; then
-    # Chrome needs hicolor-icon-theme to configure/install
-    install hicolor-icon-theme
+if [ "${DISTROAKA:-"$DISTRO"}" = 'arch' ]; then
+    install aur:google-chrome$CHANNEL
```

Comment:
There are a number of reasons to prefer AUR:
 - PKGBUILD does a number of fixups specific to Arch (https://aur.archlinux.org/packages/go/google-chrome/PKGBUILD).
 - Also, it's a PKGBUILD, so installed files are maintained in pacman database and the package can easily be removed as needed (we want to avoid "cruft": files that are not owned by any packages).
 - Finally, AUR package version is bumped after each Chrome update, so `aurcompile` can update it only when needed (on Ubuntu/Debian, Chrome adds an apt repository to address that issue).

It would not be practical to maintain our own PKGBUILD in crouton (e.g. bumping versions as needed, copy-pasting hacks from the "official" PKGBUILD), and I don't think `google-chrome` will ever get in official repository due to licensing issues. However AUR has security issues so it's not ideal either.

All in all, I'd rather drop support, instead of building something complicated that will surely break occasionally, and eat our limited resources as maintainers: After all, neither ArchLinux nor Google really supports Chrome on Arch (and, to be honest, Chrome in crouton makes little sense for most use cases). And we already do not have support on ARM, anyway...

If an Arch user really wants Chrome, it is very easy to fetch and build the PKGBUILD from AUR (the target file only does that, no crouton hack is needed!). So we push the responsibility of taking a (small) security risk onto the user.